test/ffmpeg-qsv/encode/avc.py

```diff
@@ -201,6 +201,12 @@ def test_r2r(self, case, gop, slices, bitrate, fps, quality, refs, profile):
     self.init(spec_r2r, case, gop, slices, bitrate, fps, quality, refs, profile)
     self.encode()
 
+  @slash.parametrize(*gen_avc_vbr_lp_parameters(spec, ['high', 'main']))
```

Comment:
We probably want to create a new, TCBRC variant specific case generator for this.